Answers.md

```diff
@@ -1 +1,6 @@
 
+1. the difference between forEach and map is that forEach doesnt actually return anything it just uses a function for each element in an array map will return a new array of the same size.
+2.the difference between a function and a method is that functions are defined outside of classes and methods are inside of classes.
+3.closures are created every time an enclosing outer function is called.
```

Comment:
Yes but remember about scope and being able to access certain variables inside of a functions scope and looking out to find the variable it is using, if it cannot find it inside